regression/cbmc/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
-  set(gcc_only "-X gcc-only")
+  set(gcc_only -X gcc-only)
```

Comment:
Should have a short explanation of what the difference here is in the commit message (list vs. string).

SHA: ff54008eb8595c961bdecee02c95273dee0e2d53